backend/apps/contact/views.py

```diff
@@ -1,3 +1,10 @@
 from django.shortcuts import render
-
 # Create your views here.
+from rest_framework import generics
+from .serializer import ContactSerializers
+from .models import Contact
+
+
+class ContactListView(generics.ListAPIView):
+    queryset = Contact.objects.all()  
+    serializer_class = ContactSerializers
```

Comment:
Contact can be take `Post` method too can be revize please.

Reply:
[c0a98ff](https://github.com/RythmSoft-Co/congenial-dollop/pull/6/commits/c0a98ff56662524fbbb5416417cbed423bd91970)

i Did it